include/fs_utils.h

```diff
@@ -212,4 +212,8 @@ void set_dos_file_time(const NativeFileHandle handle, const uint16_t date, const
 
 bool delete_native_file(const std_fs::path& path);
 
+// Simple file/directory removal routines, without MS-DOS compatibility hacks
+bool simple_delete_file(const std_fs::path& path);
+bool simple_remove_dir(const std_fs::path& path);
```

Comment:
Drop the simple prefix? Is there any harm in just using `delete_native_file`?

Reply:
OK, dropped the prefix.

`delete_native_file` and `local_drive_remove_dir` contains workarounds to behave more similar to MS-DOS functions - I'm not sure if we could safely use them in all cases, especially within the `drive_overlay.cpp`.